source/appModules/mmc.py

```diff
@@ -20,10 +22,36 @@ def _get_focusRedirect(self):
 
 
 class MMCTableCell(IAccessible):
+	""" Cell and rowheader makes no sense for these controls. Mapping them to list items has added benefit
+	of suppressing selected. """
+	role = controlTypes.ROLE_LISTITEM
+
 	def event_selection(self):
 		if self.parent.hasFocus and api.getFocusObject() != self:
 			eventHandler.executeEvent("gainFocus", self)
 
+	def _get_positionInfo(self):
+		""" When 'Guess object position info when unavailable' is enabled
+		these controls reports very strange information such as 65537 of 12, especially in braille.
+		Disable reporting of position info all together. """
+		return None
+
+
+class emptyToolTip(ToolTip):
+	previousToolTipText = ''
+
+	def _get_name(self):
+		""" ToolTips appearing when hovering mouse over graphical view have empty name,
+		but it can be retrieved via display model. """
+		return self.displayText
+
+	def event_show(self):
+		# Stop repeating the same tooltip over and over again.
+		toolTipText = self.displayText
+		if toolTipText != emptyToolTip.previousToolTipText:
+			emptyToolTip.previousToolTipText = toolTipText
+			return super(emptyToolTip, self).event_show()

```

Comment:
Events don't return anything. Also, notice the new, easier super call that is supported in Python 3.

```suggestion
			super().event_show()
```